lib/bolt/bolt_option_parser.rb

```diff
@@ -1092,7 +1092,7 @@ def initialize(options)
         @options[:log] = { 'console' => { 'level' => level } }
       end
       define('--clear-cache',
-             "Clear plugin cache before executing.") do |_|
+             "Clear plugin cache and plan/task cache before executing.") do |_|
```

Comment:
```suggestion
             "Clear plugin, plan, and task caches before executing.") do |_|
```

Or "Clear cache files before executing.", though I think the extra specificity is helpful.